modules/nw-ovn-kubernetes-live-migration-about.adoc

```diff
@@ -0,0 +1,88 @@
+
+* OpenShift SDN does not support IPsec. After migration, cluster administrators can enable IPsec.
+
+* OpenShift SDN does not support IPv6. Consequently, live migration to OVN-Kubernetes with IPv6 is not supported. Cluster administrators can enable dual-stack if necessary after migration is completed.
+
+* The cluster MTU is the MTU value for pod interfaces. It is always less than your hardware MTU to account for the cluster network overlay overhead. The overhead is 100 bytes for OVN-Kubernetes and 50 bytes for OpenShift SDN.
++
+During the live migration, both OVN-Kubernetes and OpenShift SDN run in parallel. OVN-Kubernetes manages the cluster network of some nodes, while OpenShift SDN manages the cluster network of others. To ensure that cross-CNI traffic remains functional, the Cluster Network Operator updates the routable MTU to ensure that both CNIs share the same overlay MTU. As a result, after the migration has completed, the cluster MTU is 50 bytes less.
+
+* Some parameters of OVN-Kubernetes cannot be changed after installation. The following parameters can be set before starting the live migration:
+
+** `InternalTransitSwitchSubnet`
+** `internalJoinSubnet`
+
+* Unless otherwise configured, OVN-Kubernetes uses the following IP address ranges:
+** `100.64.0.0/1`. This IP address range is used for `internalJoinSubnet` of OVN-Kubernetes by default. If this IP address range is already in use, enter the following command to update it to `100.63.0.0/16`:
++
+[source,terminal]
+----
+$ oc patch network.operator.openshift.io cluster --type='merge' -p='{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"ipv4":{"internalJoinSubnet": "100.63.0.0/16"}}}}}'
+----
+** `100.88.0.0/16`. This IP address range is used for `internalTransSwitchSubnet` of OVN-Kubernetes by default. If this IP address range is already in use, enter the following command to update it to `100.99.0.0/16`:
++
+[source,terminal]
+----
+$ oc patch network.operator.openshift.io cluster --type='merge' -p='{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"ipv4":{"internalTransitSwitchSubnet": "100.99.0.0/16"}}}}}'
+----
+
+* In most cases, the live migration is independent to the secondary interfaces of pods created by the Multus CNI plugin. However, if these secondary interfaces were set up on the default network interface controller (NIC) of the host, for example, using macvlan or bridge interfaces with the default NIC as the control node, OVN-Kubernetes might encounter malfunctions. Users should remove such configurations before proceeding with the live migration.
+
+* When there are multiple NICs inside of the host, and the default route is not on the interface that has the Kubernetes NodeIP, you must use the offline migration instead.
+
+* All DaemonSets in the `openshift-sdn` namespace, which are not managed by the Cluster Network Operator (CNO), must be removed. These unmanaged DaemonSets can cause the migration status to remain incomplete if not properly handled.
```

Comment:
About the statement of `All DaemonSets in the openshift-sdn namespace, which are not managed by the Cluster Network Operator (CNO), must be removed`, should we say must be removed after the migration?

Reply:
Any unmanaged daemonset shall be removed before the migration.

Reply:
Revised to: 

```suggestion
* All `DaemonSet` objects in the `openshift-sdn` namespace, which are not managed by the Cluster Network Operator (CNO), must be removed before initiating the live migration. These unmanaged DaemonSets can cause the migration status to remain incomplete if not properly handled.
* ```